test/Dockerfile

```diff
@@ -1,22 +1,18 @@
-FROM python:3.5.2
+FROM hashicorp/terraform as terraform
 
-ENV TERRAFORM_VERSION 0.11.8
+FROM python:3.6
```

Comment:
Please upgrade to 3.8, unless you have a good reason not too. It works just fine for me.

Reply:
Good idea.